src/command_modules/azure-cli-eventgrid/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+0.1.7
++++++
+* Consolidated event subscription operations into a single category, and added support for update commands.
```

Comment:
Since consolidating these commands results in a breaking change for your users (albeit preview users) you should be more specific here. Call out which update commands were added, and call out each of the old commands and the corresponding new command.  Annotate that with BC to denote the breaking change. For example:

```
* Added `eventgrid topic update` command
* BC: `eventgrid blah foo` command renamed `eventgrid do-foo`.  
```

Since you're in preview breaking changes are okay, but it will help your existing preview customers understand what broke, why, and what to update to.